### ISSUE TITLE
Cannot use with multiple instances

### ISSUE THREAD

**Issue by joedawson** [clampy-js/vue-clampy#3]
When using this directive on multiple instances, each instance is replaced with the content of the first instance.

For example the content in `#two` and `#three` here is replaced with the clamp content from `#one`.

```html
<p id="one" v-clampy="2">First instance lorem ipsum dolor sit amet, consectetur adipisicing elit. Incidunt, mollitia sed tempore doloribus, tenetur ducimus voluptatem aliquid sunt accusamus. Totam eaque asperiores, reprehenderit debitis, molestias dolore quasi itaque magnam adipisci!</p>
<p id="two" v-clampy="2">Second instance lorem ipsum dolor sit amet, consectetur adipisicing elit. Incidunt, mollitia sed tempore doloribus, tenetur ducimus voluptatem aliquid sunt accusamus. Totam eaque asperiores, reprehenderit debitis, molestias dolore quasi itaque magnam adipisci!</p>
<p id="three" v-clampy="2">Third instance lorem ipsum dolor sit amet, consectetur adipisicing elit. Incidunt, mollitia sed tempore doloribus, tenetur ducimus voluptatem aliquid sunt accusamus. Totam eaque asperiores, reprehenderit debitis, molestias dolore quasi itaque magnam adipisci!</p>
```

**Comment by joedawson** [comment]
Here's a screenshot, the 2nd video here (APROBLEMM - BL@CKBOX) has a description and so does the videos following that. You can see they're using the BL@CKBOX description. 

![screen shot 2018-07-30 at 08 39 13](https://user-images.githubusercontent.com/1009696/43383825-1edde03a-93d4-11e8-8b9c-873026204012.png)

**Comment by alexandremoore** [comment]
@JoeDawson this should be fixed in 1.0.3.

**Comment by joedawson** [comment]
Hi @alexandremoore,

Thanks - I can confirm this has resolved my issue now.

Appreciate the help!
